src/resources/editor/tools/yaml/yaml-intelligence-resources.json

```diff
@@ -8529,6 +8529,14 @@
       },
       "description": "The bibliography style to use (e.g. `\\bibliographystyle{dinat}`) when using `natbib` or `biblatex`."
     },
+    {
```

Comment:
This actually needs to be added in the YAML (then this file is generated w/ `quarto build-js`). i.e. the change should be in `document-references.yml`

Reply:
Thanks I think I did the right thing now. I need to remember how to do this